packages/rs-drive-abci/src/execution/validation/state_transition/state_transitions/data_contract_create/structure/v0/mod.rs

```diff
@@ -17,6 +20,12 @@ impl DataContractCreatedStateTransitionStructureValidationV0 for DataContractCre
         &self,
         platform_version: &PlatformVersion,
     ) -> Result<SimpleConsensusValidationResult, Error> {
+        if self.data_contract().document_schemas().is_empty() {
+            return Ok(SimpleConsensusValidationResult::new_with_error(
+                DataContractEmptySchemaError::new(self.data_contract().id().clone()).into(),
+            ));
+        }
+
         // Validate data contract
         let result = DataContract::try_from_platform_versioned(
```

Comment:
Actual schema validation is happening here for each document type https://github.com/dashpay/platform/blob/10954c2355903c42ff17a2c3f7f57a5f674621ce/packages/rs-dpp/src/data_contract/document_type/class_methods/try_from_schema/mod.rs#L11

We need to validate here that we document types are defined https://github.com/dashpay/platform/blob/10954c2355903c42ff17a2c3f7f57a5f674621ce/packages/rs-dpp/src/data_contract/document_type/class_methods/create_document_types_from_document_schemas/mod.rs#L35